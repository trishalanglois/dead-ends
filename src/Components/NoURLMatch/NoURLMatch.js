import React from 'react';
import './NoURLMatch.scss';

export const NoURLMatch = () => {
  return (
    <main className="no-url-container">
      <h2 className="no-url-match-message">This page has descended into darkness.  Please return home.</h2>
    </main>
  )
}

export default NoURLMatch;
