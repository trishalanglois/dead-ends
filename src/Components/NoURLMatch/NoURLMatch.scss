@import '../../scss/variables';

.no-url-container {
  background-color: $dark-color;
  text-align: center;
  height: 100vh;
}

.no-url-match-message {
  font-family: $accent-font;
  color: $light-grey;
  font-size: 2em;
  font-weight: 200;
  height: 100vh;
  padding-top: 5em;
  margin-top: -1em;
}
