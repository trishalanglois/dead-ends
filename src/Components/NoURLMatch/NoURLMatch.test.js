import React from 'react';
import { render } from '@testing-library/react';
import NoURLMatch from './NoURLMatch';
import { shallow } from 'enzyme';


describe('NoURLMatch', () => {
  it('should match the snapshot', () => {
    let wrapper = shallow(<NoURLMatch />)

    expect(wrapper).toMatchSnapshot();
  })
})
