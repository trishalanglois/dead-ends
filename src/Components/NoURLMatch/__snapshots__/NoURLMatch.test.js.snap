// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoURLMatch should match the snapshot 1`] = `
<main
  className="no-url-container"
>
  <h2
    className="no-url-match-message"
  >
    This page is a dead end.  Please return home.
  </h2>
</main>
`;
